crates/e2e/tests/e2e/solver_competition.rs

```diff
@@ -139,20 +139,20 @@ async fn solver_competition(web3: Web3) {
         .await
         .unwrap();
 
-    assert!(competition.common.solutions.len() == 2);
+    assert!(competition.solutions.len() == 2);
 
     // Non winning candidate
-    assert!(competition.common.solutions[0].ranking == 2);
-    assert!(!competition.common.solutions[0].is_winner);
+    assert!(competition.solutions[0].ranking == 2);
+    assert!(!competition.solutions[0].is_winner);
     // Winning candidate
-    assert!(competition.common.solutions[1].ranking == 1);
-    assert!(competition.common.solutions[1].is_winner);
+    assert!(competition.solutions[1].ranking == 1);
+    assert!(competition.solutions[1].is_winner);
```

Comment:
I would add some asserts regarding the new fields in v2: `reference_scores` (we should be able to assert the exact value) and `tx_hash` (if not possible to have the exact value, at least that it's not empty)